packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts

```diff
@@ -160,9 +165,15 @@ export class OrgList implements vscode.Disposable {
     }
   }
 
+  public async getDefaultUsernameOrAlias() {
```

Comment:
Where is this method getting used? It looks like it has the same functionality as the `getDefaultDevHubUsernameOrAlias(`) in this class

Reply:
Good catch, that was junk i forgot to take out